test/tests.js

```diff
@@ -0,0 +1,16 @@
+import isYes from '../src/is-yes.js';
+const test = QUnit.test;
+
+test('returns true if yes', function(assert) {
```

Comment:
You have this individual test set up perfectly! You'll need to add more tests thought to test all cases for the function - test what it returns if no, if the user enters something other than yes/no, etc.